docker-compose-osx.yml

```diff
@@ -0,0 +1,90 @@
+version: "3"
```

Comment:
Instead of duplicating the entire docker-compose.yml it's possible to layer additional config options on top of the base using "docker-compose -f docker-compose.yml -f docker-compose-osx.yml" If you used that this file could be reduced to something like this which might be easier to maintain.
```version: '3'
services:
   sidekiq:
      volumes:
         - api-sync:/clinwiki:delegated
```

Reply:
Are we okay with all the `:delegated` options in the canonical `docker-compose.yml`? If so, then I can make this change.

Reply:
Unless it's different on Mac the volumes live on the native linux (ext4) filesystem and don't suffer the same problem as a filesystem shared with the host OS so I'm not sure if :delegated is having much impact.  On the other hand I don't think it hurts either since those are only written from the container.

I'm a little wary of having :delegated on the /clinwiki mount since it's a common workflow to edit the /front and run `yarn start` from outside.  I think the Code The Dream guys do this on Mac also.